### PR TITLE
Multiple branches from same commit

### PR DESCRIPTION
**Pull request by nicoespeon**

### CODE REVIEW

**Comment by joneserick** [comment]
@nicoespeon is this PR still valid? I'm really interested in using this feature of multiple branches from the same commit.

**Comment by nicoespeon** [comment]
@joneserick well it is because it only added the tests showing that the feature is missing.

You can see that there's no `develop` branch showing up here: https://www.chromaticqa.com/snapshot?appId=5c9632d9ead30500207bb576&id=5cb3b3a140be8b0020b2216b

image

There should be.

The goal would be to make the tests pass.

File: packages/stories/src/gitgraph-js/1-basic-usage.stories.tsx [hunk 1]
```diff
@@ -226,6 +226,21 @@ storiesOf("gitgraph-js/1. Basic usage", module)
...
   ))
+  .add("multiple branches at the same commit", () => (
+    <GraphContainer>
+      {(graphContainer) => {
+        const gitgraph = createGitgraph(graphContainer, {
+          generateCommitHash: createFixedHashGenerator(),
+        });
+
+        const master = gitgraph.branch("master").commit("one");
+        const feat1 = gitgraph.branch("feat/1");
```
Comment (nicoespeon) [inline packages/stories/src/gitgraph-js/1-basic-usage.stories.tsx:237]:
```suggestion
        const feat1 = gitgraph.branch("feat/1-trigger-ci");
```

**Comment by iliakan** [comment]
Will multiple branches show correctly, if the structure is imported via `gitgraph.import()`?

**Pull request closed without merging**